api/kubevirt.io/resource/pci/v1alpha1/pciclaimparameters.go

```diff
@@ -0,0 +1,33 @@
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// PciClaimParametersSpec is the spec for the PciClaimParameters CRD.
+type PciClaimParametersSpec struct {
+	DeviceName string `json:"deviceName"`
```

Comment:
Right now, I don't think we have any specification. Could we use the Name in the `ObjectMetadata`?

Reply:
I think it would be better to keep it separate from name in spec
kubernetes resource names has its own validation like its cannot contain slashes ('/') etc , for its own API , which would prevent us from doing something like 
```
apiVersion: pci.resource.kubevirt.io/v1alpha1
kind: PciClaimParameters
metadata:
  name: devices.kubevirt.io/nvme
  namespace: pci-nvme-test1
```
We would have map to some other object like config map or use label/annotation etc if we want to remove spec

Reply:
I don't think we will need the slash, that name is typical for the device plugin. DRA is much more flexible. Anyway, as the comment above, right now we can leave as it is.